### ISSUE TITLE
Source codes in src/input

### ISSUE THREAD

**Issue by bingao** [openrsp/openrsp#40]
Hi @bast and @ringholm 

Do you know if these codes (in `src/input`) are still used or need to be kept for some time?

Please also update `doc/developer/files_and_directories.rst` for these files.

**Comment by ringholm** [comment]
I think that they belong to the Dalton version of OpenRSP and are not used currently. I have for now removed the reference to them in `doc/developer/files_and_directories.rst`.

**Comment by bingao** [comment]
Since they are not used currently, do we still need to keep them in repository? If yes, I think we need to know what they will be used for.

**Comment by bast** [comment]
If they are not used, I suggest to remove them.

**Comment by ringholm** [comment]
Fine with me to remove them.

**Comment by bingao** [comment]
Hi, I have removed these files and committed to the branch `documentation-second-pass`. Once this branch is merged into `master`, we can close this issue.

**Comment by bast** [comment]
Please auto-close issues in commit messages so that we don't have to remember to close issues and so that issues and code changes are cross-referenced.

**Comment by ringholm** [comment]
Now fixed, closing.
